### PR TITLE
import: add download wait duration metrics (#9373)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #9373 to release-5.0-rc

---

### What problem does this PR solve?

Add download wait duration metrics.

### Related changes

- Need to cherry-pick to the release branch

### Release note 

- No release note

### CODE REVIEW

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -64,7 +64,11 @@
   "gnetId": null,
   "graphTooltip": 1,
   "id": null,
+<<<<<<< HEAD
   "iteration": 1589260003867,
+=======
+  "iteration": 1609133726628,
+>>>>>>> a69c8eaa8... import: add download wait duration metrics (#9373)
```
Comment (kennytm) [inline metrics/grafana/tikv_details.json:71]: merge conflict

**Review by kennytm — COMMENTED** [review]
rest LGTM @overvenus

**Comment by glorv** [comment]
LGTM

**Review by 3pointer — APPROVED** [review]
LGTM

**Pull request merged**